bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -101,6 +101,31 @@
     public final static String CLIENT_ROLE_SYSTEM = "system";
 
     /**
+     * Use SSL while talking to Bookies
+     */
+    protected final static String USE_SSL = "useSSL";
+
+    /**
+     * Verify SSL certificates
+     */
+    protected final static String VERIFY_SSL_CERTIFICATES = "verifySSLCertificates";
+
+    /**
+     * Enable client authentication using an SSL certificate
+     */
+    protected final static String CLIENT_SSL_AUTHENTICATION = "clientSSLAuthentication";
+
+    /**
+     * Key store which holds the client-side SSL certificate. The certificate must be in PKCS12 format
```

Comment:
This comes from the original patch from @ivankelly. In production I usually use PKCS12. I can make a new option for the format. Do you think it wild be useful?